src/os/layer/preset/presetmenubutton_shim.js

```diff
@@ -0,0 +1,16 @@
+goog.provide('os.layer.preset.PresetMenuButtonCtrl');
```

Comment:
We shouldn't need a shim file for this as its a new directive and only required in one place.